tests/Load/utils/scenarioUtils.js

```diff
@@ -0,0 +1,99 @@
+import ScenariosBuilder from './scenariosBuilder.js';
+import ThresholdsBuilder from './thesholdsBuilder.js';
+
+export default class ScenarioUtils {
+    constructor(utils, scenarioName) {
+        this.utils = utils;
+        this.config = utils.getConfig();
+        this.smokeConfig = this.config.endpoints[scenarioName].smoke;
+        this.averageConfig = this.config.endpoints[scenarioName].average;
+        this.stressConfig = this.config.endpoints[scenarioName].stress;
+        this.spikeConfig = this.config.endpoints[scenarioName].spike;
+        this.setupTimeout =
+            this.config.endpoints[scenarioName].setupTimeoutInMinutes + 'm';
+        this.delay = this.config.delayBetweenScenarios;
+        this.averageTestStartTime = 0;
+        this.stressTestStartTime = 0;
+        this.spikeTestStartTime = 0;
+    }
+
+    getOptions() {
+        return {
+            setupTimeout: this.setupTimeout,
+            insecureSkipTLSVerify: true,
+            scenarios: this.getScenarios(),
+            thresholds: this.getThresholds()
+        };
+    }
+
+    getScenarios() {
+        const scenariosBuilder = new ScenariosBuilder();
+        const scenarioFunctions = {
+            'run_smoke': this.addSmokeScenario.bind(this, scenariosBuilder),
+            'run_average': this.addAverageScenario.bind(this, scenariosBuilder),
+            'run_stress': this.addStressScenario.bind(this, scenariosBuilder),
+            'run_spike': this.addSpikeScenario.bind(this, scenariosBuilder)
+        };
+
+        for (const key in scenarioFunctions) {
+            if (this.utils.getCLIVariable(key) !== 'false') {
```

Comment:
why 'false' is string? need to be boolean

Reply:
why this case is not fixed?

Reply:
Because we receive a string from getCLIVariable(key), so, we check it for not being equal to 'false'